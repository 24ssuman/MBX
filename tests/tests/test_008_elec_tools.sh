#!/bin/bash

filename=$(basename "$0")
filename="${filename%.*}"

../../build/stage/usr/local/clusters_ultimate/bin/elec_tools-test inputs/${filename}.nrg > output

if diff expected/${filename}.out output &> /dev/null ; then
  echo "TEST $filename PASSED" 
  rm -f input.nrg output*
  exit 0
else
  echo "TEST $filename FAILED"
  echo "OUTPUT..."
  cat output
  echo
  echo "EXPECTED"
  cat expected/${filename}.out
  rm -f input.nrg output*
  exit 1
fi
