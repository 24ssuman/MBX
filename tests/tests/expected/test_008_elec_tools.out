M-sites:
	X		Y			Z
       O 1.29053e+00	4.40150e+00	3.83867e-02	
       H 1.40062e+00	5.02504e+00	7.56240e-01	
       H 3.38249e-01	4.22779e+00	-1.13894e-02	
    virt 1.11085e+00	4.49747e+00	1.80923e-01	
       O 1.27803e+00	1.60669e+00	2.03625e-01	
       H 1.60463e+00	2.51914e+00	1.82890e-01	
       H 1.85519e+00	1.11079e+00	-3.76182e-01	
    virt 1.47085e+00	1.69556e+00	7.54974e-02	
       O-1.13520e+00	3.01117e+00	-3.01867e-02	
       H-1.75524e+00	2.83899e+00	-7.38120e-01	
       H-5.23152e-01	2.26042e+00	-2.52638e-02	
    virt-1.13690e+00	2.81426e+00	-1.80176e-01	

Charges:
       O 0.00000e+00
       H 1.05271e+01
       H 1.04991e+01
    virt-2.10262e+01
       O 0.00000e+00
       H 1.05078e+01
       H 1.05370e+01
    virt-2.10448e+01
       O 0.00000e+00
       H 1.05434e+01
       H 1.05161e+01
    virt-2.10595e+01

Polfac: 
       O 1.31000e+00
       H 2.94000e-01
       H 2.94000e-01
    virt 0.00000e+00
       O 1.31000e+00
       H 2.94000e-01
       H 2.94000e-01
    virt 0.00000e+00
       O 1.31000e+00
       H 2.94000e-01
       H 2.94000e-01
    virt 0.00000e+00

Pols: 
       O 1.31000e+00
       H 2.94000e-01
       H 2.94000e-01
    virt 1.31000e+00
       O 1.31000e+00
       H 2.94000e-01
       H 2.94000e-01
    virt 1.31000e+00
       O 1.31000e+00
       H 2.94000e-01
       H 2.94000e-01
    virt 1.31000e+00
